From e7ae35594437badfcd255f009baa5f2ac1bda6a0 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 23 Sep 2014 14:56:12 -0400
Subject: [redhat] configs: enable bochs kms driver

Message-id: <1411484172-28849-10-git-send-email-kraxel@redhat.com>
Patchwork-id: 96484
O-Subject: [RHEL-7.1 kernel PATCH 9/9] enable bochs kms driver
Bugzilla: 1043379
RH-Acked-by: Dave Airlie <airlied@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_DRM_BOCHS b/redhat/configs/generic/CONFIG_DRM_BOCHS
new file mode 100644
index 0000000..8dff9db
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_DRM_BOCHS
@@ -0,0 +1 @@
+CONFIG_DRM_BOCHS=m
-- 
1.7.1